scripts/print_dev_env.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/sh
```

Comment:
Can we change the name of the script, to something like 'output_env.sh'. This may be run on test machines as well as development machines.